ckanext/datatablesview/templates/datatables/datatables_form.html

```diff
@@ -14,13 +14,20 @@
   value='True',
   checked=data.export_buttons,
   ) }}
-  {{ form.checkbox(
-    'col_reorder',
-    id='field-col_reorder',
-    label=_('Column reorder'),
-    value='True',
-    checked=data.col_reorder,
-    ) }}
+{{ form.checkbox(
+  'col_reorder',
+  id='field-col_reorder',
+  label=_('Column reorder'),
+  value='True',
+  checked=data.col_reorder,
+  ) }}
+{{ form.checkbox(
+  'fixed_columns',
+  id='field-fixed_columns',
+  label=_('Fixed left column'),
+  value='True',
+  checked=data.fixed_columns,
```

Comment:
Is this setting fixed columns to be true by default? Your PR says it should be off by default so I'm just double checking the behavior

Reply:
When the checkbox is checked the value is 'True'. The default is set in plugin.py line 52